HW-11/homework11.py

```diff
@@ -0,0 +1,59 @@
+import json
+data = {
+    'Frodo': '12345678',
+    'Sam': '87654321',
+    'Mari': '18273645',
+
+}
+
+with open(r'C:\HW\HomeWork\HW-11\file', 'w+') as file:
```

Comment:
Here and everywhere else it is better to use relative path instead of the absolute since the absolute will work only in your computer. Relative path looks like this: `open('file', 'w+') as file:`